_includes/live-demos/spellcheckerpro/index.js

```diff
@@ -4,5 +4,16 @@ tinymce.init({
   toolbar: 'spellchecker language spellcheckdialog',
   height: 500,
   spellchecker_language: 'en',
+  content_langs: [
+    { title: 'English (US)', code: 'en_US' },
+    { title: 'English (US Medical)', code: 'en_US', customCode: 'en_US-medical' },
```

Comment:
how does our server work for Us Medical with the code being the same with en_US? do we also pass the customCode to our servers?

Reply:
hazza the docs below answers this :)